app/controllers/locations/floors_controller.rb

```diff
@@ -0,0 +1,62 @@
+require_dependency "locations/application_controller"
+
+module Locations
+  class FloorsController < ApplicationController
+    before_action :set_floor, only: [:show, :edit, :update, :destroy]
+
+    # GET /floors
+    def index
+      @floors = Floor.all
+    end
+
+    # GET /floors/1
+    def show
+    end
+
+    # GET /floors/new
+    def new
+      @floor = Floor.new(floor_params)
```

Comment:
Don't think you need the Floor.new(floor_params) here, just Floor.new should suffice.